packages/react-components/react-list-preview/src/components/ListItemButton/ListItemButton.types.ts

```diff
@@ -1,7 +1,8 @@
+import { Button } from '@fluentui/react-button';
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
 export type ListItemButtonSlots = {
-  root: Slot<'div'>;
+  root: Slot<typeof Button>;
```

Comment:
ListItemButton is not used anywhere - what is it?

Reply:
It's a component provided to be used in case the list item should be actionable (and only have one primary action). This is pretty much a button with some predefined styles, originally I had a button slot (on which you would attach the action) but after a discussion with Shift he preferred this idea. You can see it in use in the stories.